src/Tribe/Event_Repository.php

```diff
@@ -57,18 +106,18 @@ public function filter_by_cost( $value, $operator = '=', $symbol = null ) {
 			'NOT BETWEEN',
 			'IN',
 			'NOT IN',
-		) ) ) {
+		] ) ) {
```

Comment:
:no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: ac2d61d88183bc1209b9295de12dbfad</sub>